src/Commands/GenerateDocumentation.php

```diff
@@ -86,6 +86,14 @@ public function getDocConfig(): DocumentationConfig
         return $this->docConfig;
     }
 
+    protected function runBootstrapHook()
+    {
+        if (is_callable(Globals::$__bootstrap)) {
+            c::info("Running `bootstrap()` hook...");
```

Comment:
I'd remove this line. Running the bootstrap hook is something that should always happen. It's also done very close to the start, so chances of it not running because of some odd bug are low. No need to log it every time. Or at most, a `debug`.